buildconfig/Jenkins/buildscript

```diff
@@ -353,6 +354,8 @@ fi
 # User Documentation
 ###############################################################################
 if [[ ${DO_DOCTESTS_USER} == true ]]; then
+  # use default configuration
+  rm -f $userprops_file
```

Comment:
Is it a problem that it is now missing, as apposed to empty, before removing line 346?

Reply:
It shouldn't be as the doc tests spin up MantidPlot and that has a single initialization point for mantid so there won't be a race-condition with creating the file.